plugins/fields.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: fields
 spec:
   platforms:
-  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.2/kubectl-fields-Darwin-x86_64.tar.gz
-    sha256: b46f1cc4c67146899a8e01d3f2048dd8bbde7ff70230009a81cc0f3e7e4c7c3e
+  - uri: https://github.com/rewanth1997/kubectl-fields/releases/download/v1.2.3/kubectl-fields-Darwin-x86_64.tar.gz
+    sha256: 2b07be546f1b5f69f718b99d8c874336895850021fb905175e304ba15f71ccb4
     bin: kubectl-fields
     files:
     - from: "*"
```

Comment:
No need to act: `files: { from: '*', to: '.' }` is now the default, so you don't need to spell it out.